src/Session.ts

```diff
@@ -887,7 +835,7 @@ export class Session implements IIndexable {
 
 	winstonTakePile() {
 		const s = this.draftState as WinstonDraftState;
-		if (!this.drafting || !s || !(s instanceof WinstonDraftState)) return false;
```

Comment:
Why did you remove these checks?
If I wanted to update this, I would use type guards rather than casting the draftState unconditionally here (adding type guards for the draft states as necessary).

Reply:
One issue is that specifically `WinstonDraftState` fields are referenced in the rest of the method, so it needs to be cast at some point.
What's the intent here? As currently written, `!(s instanceof WinstonDraftState` will always be `false` since it was explicitly cast above. So either `s` will be `undefined` or will throw a Type Error.